test/testcafe/framework/page-objects/UiDemoPageObject.js

```diff
@@ -0,0 +1,33 @@
+import BasePageObject from './BasePageObject';
+import { ClientFunction } from 'testcafe';
+
+
+export default class UiDemoPageObject extends BasePageObject {
+  constructor(t) {
+    super(t);
+  }
+
+  async scrollToNextPage() {
+    const getClientHeight = ClientFunction(() => window.document.documentElement.clientHeight);
+    const windowheight = await getClientHeight();
+    await this.t.scrollBy(0, windowheight);
+  }
+
+  async hideProgressBar() {
+    await ClientFunction(() => {
+      // the testcafe progress bar element is in the shadow dom so we cannot query it directly
+      const style = document.createElement('style');
+      style.textContent = '.progress-bar-hammerhead-shadow-ui { visibility: hidden !important; }';
+      document.head.appendChild(style);
+    })();
+  }
```

Comment:
Some of these _might_ actually be good for the `BasePageObject` or a utility function

Reply:
I think since these are now only used by the uiDemo spec outside of the `parity-v3`/`testcafe` suites, it can probably stay here for now